rubycritic.gemspec

```diff
@@ -25,9 +25,9 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "flog", "4.2.1"
   spec.add_runtime_dependency "reek", "2.0.4"
   spec.add_runtime_dependency "parser", ">= 2.2.0", "< 3.0"
+  spec.add_runtime_dependency "rake", "~> 10"
```

Comment:
I believe that `rake` should be an optional dependency here as developers who are using the RubyCritic executable and not the rake task don't have to install `rake` by default.